app/scripts/views/base.js

```diff
@@ -168,14 +168,14 @@ define(function (require, exports, module) {
         this.viewName = options.viewName;
       }
 
-      Backbone.View.call(this, options);
-
       // The mixin's initialize is called directly instead of the normal
       // override the `initialize` function because not all sub-classes
       // call the parent's `initialize`. w/o the call to the parent,
       // the mixin does not initialize correctly.
       NotifierMixin.initialize.call(this, options);
 
+      Backbone.View.call(this, options);
```

Comment:
Was this an intended change? Did it need to be initialized before something?

Reply:
Yep, `views/complete_sign_up` references `this.notifier` inside `initialize`, so I needed to call `NotifierMixin.initialize` earlier.